src/components/chart/utils/formatChartData.ts

```diff
@@ -0,0 +1,25 @@
+// @ts-ignore
```

Comment:
Why do you need this line?

Reply:
The library after (date-and-time) does not have a types